apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorScheduleSection.vue

```diff
@@ -51,21 +54,30 @@ import CollectorScheduleForm
 const props = defineProps<{}>();
 const state = reactive({
     isEditMode: false,
+    collectorScheduleHours: [] as string[],
+    updatingHours: [] as string[],
```

Comment:
suggestion: How about `collectorScheduleUTCHours` and `updatingUTCHours`? Too long?😂

Reply:
Okay ~